src/less/event.less

```diff
@@ -1,11 +1,13 @@
 @import './variables.less';
 
-.rbc-event {
+button.rbc-event {
   padding: @event-padding;
   background-color: @event-bg;
   border-radius: @event-border-radius;
   color: @event-color;
   cursor: pointer;
+  width: 100%;
```

Comment:
instead lets do `display:block;` maybe?

Reply:
`display: block` doesn't seem to apply the full width, how about both?

Reply:
Hmm `display: block` seems to ruin the `+5 more` behaviour when events can't fit vertically